src/Polly.Shared/Monkey/InjectLatencySyntax.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using Polly.Monkey;
+using Polly.Utilities;
+
+namespace Polly
+{
+    /// <summary>
+    /// Fluent API for defining an Fault Injection <see cref="Policy"/>. 
+    /// </summary>
+    public partial class Policy
+    {
+        /// <summary>
+        /// Builds a <see cref="MonkeyPolicy"/> which executes a fault if <paramref name="enabled"/> returns true and
+        /// a random number is within range of <paramref name="injectionRate"/>.
+        /// </summary>
+        /// <param name="fault">Fault Delegate to be executed</param>
+        /// <param name="injectionRate">The injection rate between [0, 1]</param>
+        /// <param name="enabled">Lambda to check if this policy is enabled in current context</param>
+        /// <returns>The policy instance.</returns>
+        public static MonkeyPolicy InjectLatency(
+            Action<Context> fault,
```

Comment:
These overloads look like they could inject any action.  To include that is fine by me, but in that case the overloads should be named `InjectCustom(...)` (or similar) to indicate that?

For an `InjectLatency(...)` overload, my preference would be would be that it takes a `TimeSpan` | `Func<Context, Timespan>` parameter, and uses Polly's `SystemClock.Sleep(...)` delegates to achieve the latency.

Should we leave `InjectLatency(...)` for @vany0114 as agreed? (if @vany0114 is still interested in contributing that).

Reply:
Oh yes For injecting any custom action they can use `Money()`.
I'll delete this one. @vany0014 can take care of this

Reply:
yeah I can take care of that, thanks guys! ...so should I wait until @mebjas finished with his PR or can I fork from his branch and start to work on it?

Reply:
@vany0114 You can start whenever you want, but if you take a branch now and @mebjas has further changes to make afterwards (which is the case), then you will have to merge his further changes up into your branch (ie rebase) and deal with any conflicts (if necessary). _Or:_ wait till @mebjas is finished, take your branch after that, and not have any potential conflicts to deal with.

Reply:
Ok, I'll wait until @mebjas finished 😃

Reply:
I'll send ASAP.